ExamenProeveUnity/Assets/Scripts/Runtime/Player/PlayerGrab.cs

```diff
@@ -22,18 +22,23 @@ private void Awake()
             _rigidbody ??= GetComponent<Rigidbody>();
         }
 
-        private void Update()
+        private void OnRightGrab(InputValue value)
         {
-            HandleMousePressed();
-            HandleMouseRelease();
+            if (mouseType != MouseType.Right) return;
```

Comment:
new input system uses controller triggers now too
change name to something like handType